Project/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAgent.cs

```diff
@@ -2,6 +2,8 @@
 using Unity.MLAgents;
 using Unity.MLAgentsExamples;
 using Unity.MLAgents.Sensors;
+using Unity.MLAgents.Extensions.Sensors;
```

Comment:
Will revert changes to this scene before committing, I was just using it as a testbed.